opus/application/apps/results/views.py

```diff
@@ -1155,11 +1155,13 @@ def api_get_product_types_for_opus_id(request, opus_id):
     cursor.execute(sql, values)
 
     results = cursor.fetchall()
-    product_types = [{'category': x[0],
-                      'product_type': x[1],
-                      'description': x[2],
-                      'version_number': x[3],
-                      'version_name': x[4]} for x in results]
+    for x in results:
+        prod_type = f'{x[1]}@v{x[4]}' if x[4] != 'Current' else f'{x[1]}@{x[4].lower()}'
```

Comment:
You could still do this using the list comprehension above. Also I think it would make a little more sense to say:
`f'{x[1]}@current' if x[4] == 'Current' else f'{x[1]}@v{x[4]}`